### PR TITLE
simplify multi-series test charm

### PR DESCRIPTION
**Pull request by mhilton**
remove unnecessary content from the multi-series charm and rename the charm to indicate the intent.

### CODE REVIEW

File: internal/storetesting/charm-repo/quantal/multi-series/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: multi-series
+summary: multi-series test charm
+description: Test charm that supports a number of series
+series:
+ - trusty
+ - utopic
+ - vivid
+ - wily
+provides:
```
Comment (rogpeppe) [inline internal/storetesting/charm-repo/quantal/multi-series/metadata.yaml:9]: Does it actually need to provide or require anything?

**Comment by rogpeppe** [comment]
LGTM with one question.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/charmstore/1234/
Test PASSed.

File: internal/storetesting/charm-repo/quantal/multi-series/metadata.yaml (hunk 1 above)
Comment (mhilton) [inline internal/storetesting/charm-repo/quantal/multi-series/metadata.yaml:9]: yes, there are some upload testing checks that test that these are specified as this.

**Comment by urosj** [comment]
:+1:

**Comment by mhilton** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/charmstore-merge

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci.jujugui.org:8080/job/charmstore-merge/375

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/charmstore-merge

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci.jujugui.org:8080/job/charmstore-merge/376

**Comment by mhilton** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/charmstore-merge

**Pull request merged**